### PR TITLE
AUTH-5670 added common_names to access groups to support multiple com…

### PR DESCRIPTION
**Pull request by russellkerns**
adds the common_names field to access groups to support multiple common_name rules within the same policy block

### CODE REVIEW

File: internal/sdkv2provider/resource_cloudflare_access_group_test.go [hunk 1]
```diff
@@ -21,53 +19,6 @@ func init() {
 	})
 }
 
-func testSweepCloudflareAccessGroups(r string) error {
```
Comment (jacobbednarz) [inline internal/sdkv2provider/resource_cloudflare_access_group_test.go:24]: did you intend to remove this?

Comment (russellkerns) [reply internal/sdkv2provider/resource_cloudflare_access_group_test.go:24]: Yes - I saw it was unused and was just doing some cleanup, can revert it if we want

**Comment by russellkerns** [comment]
> looks like the acceptance tests are failing at the moment too
> 
> ```
> TF_ACC=1 go test ./internal/sdkv2provider -v -run "^TestAccCloudflareAccessGroup_" -count 1 -timeout 120m -parallel 1
> === RUN   TestAccCloudflareAccessGroup_ConfigBasicZone
> --- PASS: TestAccCloudflareAccessGroup_ConfigBasicZone (18.41s)
> === RUN   TestAccCloudflareAccessGroup_ConfigBasicAccount
> --- PASS: TestAccCloudflareAccessGroup_ConfigBasicAccount (13.01s)
> === RUN   TestAccCloudflareAccessGroup_Exclude
> --- PASS: TestAccCloudflareAccessGroup_Exclude (14.48s)
> === RUN   TestAccCloudflareAccessGroup_Require
> --- PASS: TestAccCloudflareAccessGroup_Require (7.88s)
> === RUN   TestAccCloudflareAccessGroup_FullConfig
> --- PASS: TestAccCloudflareAccessGroup_FullConfig (7.59s)
> === RUN   TestAccCloudflareAccessGroup_WithIDP
> --- PASS: TestAccCloudflareAccessGroup_WithIDP (10.96s)
> === RUN   TestAccCloudflareAccessGroup_WithIDPAuthContext
> --- PASS: TestAccCloudflareAccessGroup_WithIDPAuthContext (9.78s)
> === RUN   TestAccCloudflareAccessGroup_Updated
> --- PASS: TestAccCloudflareAccessGroup_Updated (20.92s)
> === RUN   TestAccCloudflareAccessGroup_CreateAfterManualDestroy
> --- PASS: TestAccCloudflareAccessGroup_CreateAfterManualDestroy (14.63s)
> === RUN   TestAccCloudflareAccessGroup_UpdatedFromCommonNameToCommonNames
>     resource_cloudflare_access_group_test.go:351: Step 2/2 error: Check failed: Check 3/5 error: cloudflare_access_group.dtxbemrooy: Attribute 'common_names.0' not found
> --- FAIL: TestAccCloudflareAccessGroup_UpdatedFromCommonNameToCommonNames (20.04s)
> FAIL
> FAIL	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	138.415s
> FAIL
> make: *** [testacc] Error 1
> ```

Should be fixed now

**Comment by jacobbednarz** [comment]
lgtm!

```
TF_ACC=1 go test ./internal/sdkv2provider -v -run "^TestAccCloudflareAccessGroup_" -count 1 -timeout 120m -parallel 1
=== RUN   TestAccCloudflareAccessGroup_ConfigBasicZone
--- PASS: TestAccCloudflareAccessGroup_ConfigBasicZone (17.62s)
=== RUN   TestAccCloudflareAccessGroup_ConfigBasicAccount
--- PASS: TestAccCloudflareAccessGroup_ConfigBasicAccount (17.13s)
=== RUN   TestAccCloudflareAccessGroup_Exclude
--- PASS: TestAccCloudflareAccessGroup_Exclude (8.16s)
=== RUN   TestAccCloudflareAccessGroup_Require
--- PASS: TestAccCloudflareAccessGroup_Require (8.44s)
=== RUN   TestAccCloudflareAccessGroup_FullConfig
--- PASS: TestAccCloudflareAccessGroup_FullConfig (8.75s)
=== RUN   TestAccCloudflareAccessGroup_WithIDP
--- PASS: TestAccCloudflareAccessGroup_WithIDP (10.28s)
=== RUN   TestAccCloudflareAccessGroup_WithIDPAuthContext
--- PASS: TestAccCloudflareAccessGroup_WithIDPAuthContext (9.21s)
=== RUN   TestAccCloudflareAccessGroup_Updated
--- PASS: TestAccCloudflareAccessGroup_Updated (13.29s)
=== RUN   TestAccCloudflareAccessGroup_CreateAfterManualDestroy
--- PASS: TestAccCloudflareAccessGroup_CreateAfterManualDestroy (14.28s)
=== RUN   TestAccCloudflareAccessGroup_UpdatedFromCommonNameToCommonNames
--- PASS: TestAccCloudflareAccessGroup_UpdatedFromCommonNameToCommonNames (13.68s)
PASS
ok  	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	124.608s
```

File: internal/sdkv2provider/resource_cloudflare_access_group_test.go (hunk 1 above)
Comment (jacobbednarz) [reply internal/sdkv2provider/resource_cloudflare_access_group_test.go:24]: ah, this should be used by the sweeper above. let me fix that.

**Pull request merged**